app/domain/monitor/facts.rb

```diff
@@ -17,6 +18,13 @@ def count_facts_metrics!
     GovukStatsd.count(path, count)
   end
 
+  def count_facts_daily_metrics!
```

Comment:
Can we rename all the `count_blah_daily_thing` methods to something like `count_blah_for_day`? Daily suggests that we are aggregating more than a day's worth here, when in fact we're not.

Reply:
Ah I see these get renamed in a later commit.